sklearn/mixture/bayesian_mixture.py

```diff
@@ -0,0 +1,689 @@
+"""Bayesian Gaussian Mixture Model."""
+# Author: Wei Xue <xuewei4d@gmail.com>
+#         Thierry Guillemot <thierry.guillemot.work@gmail.com>
+# License: BSD 3 clause
+
+import math
+import numpy as np
+from scipy.special import digamma, gammaln
+
+from .base import BaseMixture, _check_shape
+from .gaussian_mixture import _check_precision_matrix
+from .gaussian_mixture import _check_precision_positivity
+from .gaussian_mixture import _compute_log_det_cholesky
+from .gaussian_mixture import _compute_precision_cholesky
+from .gaussian_mixture import _estimate_gaussian_parameters
+from .gaussian_mixture import _estimate_log_gaussian_prob
+from ..utils import check_array
+from ..utils.validation import check_is_fitted
+
+
+def _log_dirichlet_norm(dirichlet_concentration):
+    """Compute the log of the Dirichlet distribution normalization term.
+
+    Parameters
+    ----------
+    dirichlet_concentration : array-like, shape (n_samples,)
+        The parameters values of the Dirichlet distribution.
+
+    Returns
+    -------
+    log_dirichlet_norm : float
+        The log normalization of the Dirichlet distribution.
+    """
+    return (gammaln(np.sum(dirichlet_concentration)) -
+            np.sum(gammaln(dirichlet_concentration)))
+
+
+def _log_wishart_norm(degrees_of_freedom, log_det_precisions_chol, n_features):
+    """Compute the log of the Wishart distribution normalization term.
+
+    Parameters
+    ----------
+    degrees_of_freedom : array-like, shape (n_components,)
+        The parameters values of the Whishart distribution.
+
+    log_det_precision_chol : array-like, shape (n_components,)
+        The determinant of the cholesky decomposition of the precision matrix.
```

Comment:
This can be simplified to:

```
log_det_precision : array-like, shape (n_components,)
   The determinant of the precision matrix for each component.
```

The  determinant of the precision matrix is the same as the determinant of its Cholesky decomposition so there is no point in adding Cholesky in the name of the variable or the docstring.